cfme/configure/access_control/__init__.py

```diff
@@ -396,6 +399,33 @@ def create(self, name=None, credential=None, email=None, group=None, cost_center
         view.browser.refresh()
         return user
 
+    def delete(self, *users):
+        """
+        Delete user(s) using the Access Control EVM Users checklist
+        """
+        users = list(users)
+        delete_toolbar_txt = 'Delete selected Users'
+        full_name_header = 'Full Name'
+        delete_success_msg = 'EVM User "{}": Delete successful'
+        delete_blocked_msg = 'Default EVM User "{}" cannot be deleted'
+
+        view = navigate_to(self, 'All')
+        for user in users:
+            try:
+                find_row_kwargs = {full_name_header: user.name}
+                # This will refresh the page to find the item so you won't have multiple items
+                # selected per page
+                row = view.paginator.find_row_on_pages(view.table, **find_row_kwargs)
+                row[0].check()
```

Comment:
**OPTIONAL:** Is this the action that is deleting the user?  It's not obvious save that the next line is verifying that it was deleted.   A comment to explain that this is what is going would be helpful.